backend/mora/integrations/serviceplatformen.py

```diff
@@ -9,18 +9,52 @@
 import random
 
 import service_person_stamdata_udvidet
-
-from requests import HTTPError
+import logging
+import pathlib
+import requests
 
 from .. import util
 from .. import settings
+from .. import exceptions
+
+
+logger = logging.getLogger("flask.app")
```

Comment:
I don't think `flask.app` is a good name for a logger here. Use `__name__`?
Also, we could use the built-in logger in flask:
`flask.current_app.logger.getChild(__name__)`

Reply:
I use name - we have no flask app at that time